### PR TITLE
DIGI_30_UniqueAddOnTitle

### PR DESCRIPTION
**Pull request by bommavikas**
1)Added AddOnController
2)Added Create View

### CODE REVIEW

File: DigiOrderEnterprise/DigiOrderEnterprise/Controllers/AddOnController.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            ViewBag.ControlNviation = "CreateAddOn";
+            return View(new AddOnModel());
+        }
+
+        /// Author: Vikas Bomma
+        /// Date: 7/14/2020
+        /// Description: Inserting Unique AddOns To database
+        /// Ref. StoryId: DIGI 10,DIGI30
+        /// Reviewer: 
+        /// Version: 1.0
```
Comment (anebhargavkrishna) [inline DigiOrderEnterprise/DigiOrderEnterprise/Controllers/AddOnController.cs:27]: no need of summary in controller

Comment (bommavikas) [reply DigiOrderEnterprise/DigiOrderEnterprise/Controllers/AddOnController.cs]: Removed Summery In Controller

File: DigiOrderEnterprise/DigiOrderEnterprise/Controllers/AddOnController.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+namespace DigiOrderEnterprise.Controllers
+{
+    public class AddOnController : Controller
+    {
+        AddOnLibrary addOnLibrary = null;
+
+        public ActionResult Create()
+        {
+            ViewBag.ControlNviation = "CreateAddOn";
```
[the remark this answers is not in the document]
Comment (bommavikas) [reply DigiOrderEnterprise/DigiOrderEnterprise/Controllers/AddOnController.cs]: Rectified

File: DigiOrderEnterprise/DigiOrderEnterprise/Controllers/AddOnController.cs [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+                    TempData["Message"] = "Duplicate Title not allowed";
+                    return RedirectToAction("Create");
+                }
+            }
+            catch (Exception ex)
+            {
+                    TempData["FailureMessage"] = "Insertion Of AddOn Failed";
+                    return RedirectToAction("Create");
+            }
+            ViewBag.ControlNviation = "AddOn";
```
[the remark this answers is not in the document]
Comment (bommavikas) [reply DigiOrderEnterprise/DigiOrderEnterprise/Controllers/AddOnController.cs]: Rectified

**Pull request merged**